site/man/rabbitmq-streams.8.html

```diff
@@ -303,7 +314,155 @@ <h3 class="Ss" title="Ss" id="Stream_plugin"><a class="permalink" href="#Stream_
+      list that follows:
+    <dl class="Bl-tag">
+      <dt><var class="Ar" title="Ar">consumers</var></dt>
+      <dd>Number of consumers in the group.</dd>
+      <dt><var class="Ar" title="Ar">partition_index</var></dt>
+      <dd>The stream partition index if the stream is part of a super stream, -1
+          if it is not.</dd>
+      <dt><var class="Ar" title="Ar">reference</var></dt>
+      <dd>The group reference (name).</dd>
+      <dt><var class="Ar" title="Ar">stream</var></dt>
+      <dd>The stream the consumers are attached to.</dd>
+    </dl>
+    <div class="Pp"></div>
+    If no <var class="Ar" title="Ar">groupinfoitem</var> are specified then
+      stream, reference, partition_index, and consumers are displayed.
+    <div class="Pp"></div>
+    For example, this command displays the stream, reference, and number of
+      consumers for each group:
+    <div class="Pp"></div>
+    <div class="Bd Bd-indent lang-bash"><code class="Li">rabbitmq-streams
+      list_stream_consumer_groups stream reference consumers</code></div>
+    <div class="Pp"></div>
+  </dd>
+  <dt><a class="permalink" href="#list_stream_group_consumers"><code class="Cm" title="Cm" id="list_stream_group_consumers">list_stream_group_consumers</code></a>
+    <code class="Fl" title="Fl">--stream</code>
+    <var class="Ar" title="Ar">stream</var>
+    <code class="Fl" title="Fl">--reference</code>
+    <var class="Ar" title="Ar">reference</var>
+    [<div class="Op"><code class="Fl" title="Fl">--vhost</code>
+    <var class="Ar" title="Ar">vhost</var></div>]
+    [<div class="Op"><var class="Ar" title="Ar">consumerinfoitem
+    ...</var></div>]</dt>
+  <dd>
+    <div class="Pp"></div>
+    Lists consumers of a stream consumer group in a vhost.</dd>
+  <dt><var class="Ar" title="Ar">stream</var></dt>
+  <dd>The stream the consumers are attached to.</dd>
+  <dt><var class="Ar" title="Ar">reference</var></dt>
+  <dd>The group reference (name).</dd>
+  <dt><var class="Ar" title="Ar">vhost</var></dt>
+  <dd>The virtual host of the stream.</dd>
+</dl>
+<div class="Pp"></div>
+The <var class="Ar" title="Ar">consumerinfoitem</var> parameter is used to
+  indicate which consumer information items to include in the results. The
+  column order in the results will match the order of the parameters.
+  <var class="Ar" title="Ar">consumerinfoitem</var> can take any value from the
+  list that follows:
+<dl class="Bl-tag">
+  <dt><var class="Ar" title="Ar">connection_name</var></dt>
+  <dd>Readable name of the consumer connection.</dd>
+  <dt><var class="Ar" title="Ar">state</var></dt>
+  <dd>Consumer state; one of:
+    <ul class="Bl-bullet Bl-compact">
+      <li>active</li>
+      <li>inactive</li>
+    </ul>
+  </dd>
+  <dt><var class="Ar" title="Ar">subscription_id</var></dt>
+  <dd>The connection-scoped ID of the consumer.</dd>
 </dl>
+<div class="Pp"></div>
+If no <var class="Ar" title="Ar">consumerinfoitem</var> are specified then
```

Comment:
Same comment as above, change to "If a consumerinfoitem is not specified, then...."